### DISCUSSION TITLE
We are not able to see consumer groups in Topics. Are we missing some configuration?

### DISCUSSION THREAD

**Discussion by Thilaknath** [provectus/kafka-ui#3663] [Q&A] [answered]
Hello Team, 

Pease find attached screenshot 
![Screenshot 2023-04-13 at 8 35 52 AM](https://user-images.githubusercontent.com/3452073/231760300-8a5b6ba7-7a18-4df5-85cc-3171cb206597.png)

When we click on a topic, We are not able to see the consumers of a particular topic. Can you help us determine if we are missing some configuration?

**Comment by Thilaknath** [comment]
Also I find that the Doc links are not working in your github repository.

**Reply by Haarolean** [reply]
All docs are located [here](https://docs.kafka-ui.provectus.io/)
Can you elaborate which links are broken exactly?

**Reply by Thilaknath** [reply]
The same link you had shared above. I will verify again after a restart. I get the following `ERR_SSL_PROTOCOL_ERROR` now when i try to reach.

**Reply by Haarolean** [reply]
Probably there's a MITM for you

**Comment by Haarolean** [comment]
I need you to:
1. share the information on how you run the app (docker/jar)
2. which version you're using
3. share your config
4. check the application logs if there are any exceptions
5. check the browser's developer console if there are any failed requests/js errors

**Reply by Thilaknath** [reply]
1. We have deployed the jar (Cloud foundry run time)
2. 0.0.1-SNAPSHOT([81dc287](https://github.com/provectus/kafka-ui/commit/81dc287))
3. We pass the following config 
AUTH_TYPE (LOGIN_FORM) KAFKA_CLUSTERS_0_BOOTSTRAPSERVERS (Cluster IP's) KAFKA_SERVICE_KEY_JSON
4. I have enabled application logs now
5. No errors in console.

**Reply by Haarolean** [reply, marked as answer]
The commit you mentioned doesn't exist in our repo. If that's a fork, we can't help you with that here, unless it's reproducible with our master and/or latest release.
